modules/ztp-preparing-the-ztp-git-repository-ver-ind.adoc

```diff
@@ -42,26 +46,30 @@ The following example describes a structure using user-provided CRs for differen
 │       ├── kustomization.yaml <3>
 │       ├── ns.yaml
 │       └── source-crs/ <4>
+│         └── reference-crs/ <5>
+│         └── custom-crs/ <6>
 └── siteconfig
     ├── kustomization.yaml
     ├── version_4.13
     │   ├── helix56-v413.yaml
     │   ├── kustomization.yaml
-    │   ├── extra-manifest/ <5>
-    │   └── custom-manifest/ <6>
+    │   ├── extra-manifest/ <7>
+    │   └── custom-manifest/ <8>
     └── version_4.14
         ├── helix57-v414.yaml
         ├── kustomization.yaml
-        ├── extra-manifest/ <5>
-        └── custom-manifest/ <6>
+        ├── extra-manifest/ <7>
+        └── custom-manifest/ <8>
 
 ----
-<1> Create a top-level `kustomization` yaml file.
+<1> Create a top-level `kustomization` YAML file.
 <2> Create the version-specific directories within the custom `/policygentemplates` directory.
 <3> Create a `kustomization.yaml` file for each version.
 <4> Create a `source-crs` directory for each version to contain reference CRs from the `ztp-site-generate` container.
-<5> Create a directory within the custom `/siteconfig` directory to contain extra manifests from the `ztp-site-generate` container.
-<6> Create a folder to hold user-provided CRs.
+<5> Create the `reference-crs` directory for policy CRs extracted from the ZTP container.
+<6> Optional: Create a `custom-crs` directory for user-provided CRs.
+<7> Create a directory within the custom `/siteconfig` directory to contain extra manifests from the `ztp-site-generate` container.
+<8> Create a folder to hold user-provided CRs.
```

Comment:
This isn't my area of knowledge so take it for what you will, but the folder name is `custom-manifest` and the description of 8 says the folder is for user-provided CRs, not manifests, which was confusing to me.